backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1005,8 +1005,8 @@ databaseChangeLog:
                   name: clia_number
                   type: text
                   remarks: The CLIA number for the facility that is ordering (and presumably also conducting) the tests.
-                  constraints:
-                    nullable: false
+                  # constraints:
```

Comment:
Here and the comment below: did something change with our Liquibase? This section is suddenly failing during rollbacks where it hasn't been previously:

```
Caused by: liquibase.exception.DatabaseException: ERROR: column "clia_number" of relation "organization" contains null values [Failed SQL: (0) ALTER TABLE simple_report.organization ADD clia_number TEXT NOT NULL]
```

What it seems to be mad about here is that rows already exist in this table, and you can't impose a `NOT NULL` constraint unless you then give each of those rows a default value - otherwise, those rows would have a null value for the new column and you'd be violating the constraint you just imposed.

What I'm not sure about is what the correct default value would be here.